### PR TITLE
[NUI][ATSPI] Default accessibility support for CollectionView and rel…

### PR DESCRIPTION
**Pull request by everLEEst**
Default accessibility support for CollectionView and related classes.

### CODE REVIEW

File: src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs [hunk 1]
```diff
@@ -134,6 +134,7 @@ public CollectionView() : base()
         {
             FocusGroup = true;
             SetKeyboardNavigationSupport(true);
+            SetAccessibilityConstructor(Role.List, AccessibilityInterface.None);
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs:137]: I think you don't have to add the second argument when it's `None` value. 
But, someone else may approve it. :)

Comment (m-bialota) [reply src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs:137]: I agree with Ms. Kim, it is better to remove the second parameter :).

Comment (m-bialota) [reply src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs:137]: However please consider if the CollectionView should not support Selection interface.

Comment (everLEEst) [reply src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs]: It support items selection not it's own selection.
Count can be readable and selected item count also readable then should it be value?

Comment (aswigon) [reply src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs]: Note: AccessibilityInterface.Selection PR: https://github.com/Samsung/TizenFX/pull/3110

Comment (everLEEst) [reply src/Tizen.NUI.Components/Controls/RecyclerView/CollectionView.cs]: okay I'll wait till it merged :)

**Comment by dongsug-song** [comment]
@everLEEst 
@m-bialota 
Hello, I can assume that this PR depends on other patch which should be merged first.
so I put the **\*\*NO MERGE\*\*** label on this PR now.
**You can remove this label for yourself when this PR is ready to be reviewed and merged.**
(**\*\*NO MERGE\*\*** label has no restriction to submit PR or to edit this patch. Just for notification and marking)
Thank you.

**Review by lol-github — CHANGES_REQUESTED** [review]

File: src/Tizen.NUI.Components/Controls/RecyclerView/Item/RecyclerViewItem.Internal.cs [hunk 2]
```diff
@@ -226,6 +226,8 @@ public override void OnInitialize()
             Layout = new AbsoluteLayout();
             UpdateState();
 
+            SetAccessibilityConstructor(Role.ListItem, AccessibilityInterface.Value);
```
Comment (lol-github) [inline src/Tizen.NUI.Components/Controls/RecyclerView/Item/RecyclerViewItem.Internal.cs:229]: IMHO, RecyclerViewItem doesn't represent a numerical value from the specified range so it shouldn't implement AccessibilityInterface.Value. Removing second parameter should be OK for this class.

However, this class should override the AccessibilityCalculateStates method to support "Selected" and "Selectable" states.

File: src/Tizen.NUI.Components/Controls/RecyclerView/Item/DefaultGridItem.cs [hunk 3]
```diff
@@ -265,7 +265,10 @@ protected virtual TextLabel CreateLabel(TextLabelStyle textStyle)
         [EditorBrowsable(EditorBrowsableState.Never)]
         protected virtual ImageView CreateImage(ImageViewStyle imageStyle)
         {
-            return new ImageView(imageStyle);
+            return new ImageView(imageStyle)
+            {
+                AccessibilityHighlightable = false,
```
Comment (lol-github) [inline src/Tizen.NUI.Components/Controls/RecyclerView/Item/DefaultGridItem.cs:270]: this line might not be needed soon, as there is a change which makes ImageView not highlightable by default:
https://review.tizen.org/gerrit/#/c/platform/core/uifw/dali-toolkit/+/259931/

**Review by lol-github — CHANGES_REQUESTED** [review]

**Pull request closed without merging**